webpack.dev.config.babel.js

```diff
@@ -44,7 +44,7 @@ for (const app of appsBuildList) {
 }
 
 export default Object.assign({}, webpackConfig, {
-  devtool: 'inline-source-map',
+  devtool: 'heap-module-eval-source-map',
```

Comment:
Is this a typo? Title of the PR says "cheap" this says "heap".

Reply:
heh 🤕

Reply:
thanks!